exp.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "PocketGear",
+  "description": "PocketGear is a clean and simple Pokédex app for Pokémon GO you'll ever need, made by Pokémon enthusiasts to help you trainers get the best out of your Pokémons.",
+  "slug": "pocketgear",
+  "sdkVersion": "10.0.0",
+  "version": "1.0.0",
+  "orientation": "portrait",
+  "primaryColor": "#D85D61",
+  "iconUrl": "https://s3.amazonaws.com/exp-us-standard/icons/pocket-gear.png",
+  "loading": {
+    "iconUrl": "https://s3.amazonaws.com/exp-us-standard/icons/pocket-gear.png",
+    "hideExponentText": false
+  },
+  "androidStatusBarColor": "#000000",
+  "packagerOpts": {
+    "assetExts": [
+      "ttf", "otf"
+    ]
+  }
+}
```

Comment:
Figure out a way to,

1. Integrate Fabric for Android and iOS (without having to commit the build secret)
2. Specify separate icons for iOS and Android apps locally
3. Set the accent color to black for Android
4. Set statusbar content color to dark on Android